packages/cli/README-source.md

```diff
@@ -849,6 +844,30 @@ module.exports = {
 
 This the URL to which you should send hook data. It'll look something like `https://hooks.zapier.com/1234/abcd`. We provide it so you can make a POST request to your server. Your server should store this URL and use is as a destination when there's new data to report.
 
+For example:
```

Comment:
added the subscription function example here, since this is the relevant section